data/helpers.d/setting

```diff
@@ -230,70 +230,82 @@ ynh_webpath_register () {
 
 # Create a new permission for the app
 #
-# usage: ynh_permission_create --app "app" --permission "permission" --defaultdisallow [--urls "url" ["url" ...]]
-# | arg: app - the application id
+# usage: ynh_permission_create --permission "permission" [--urls "url" ["url" ...]]
 # | arg: permission - the name for the permission (by default a permission named "main" already exist)
-# | arg: defaultdisallow - define if all user will be allowed by default
-# | arg: urls - the list of urls for the the permission
+# | arg: urls - (optional) a list of FULL urls for the permission (e.g. domain.tld/apps/admin)
+#
+# example: ynh_permission_create --permission admin --urls domain.tld/blog/admin
 ynh_permission_create() {
-    declare -Ar args_array=( [a]=app= [p]=permission= [d]=defaultdisallow [u]=urls= )
-    local app
+    declare -Ar args_array=( [p]=permission= [u]=urls= )
     local permission
-    local defaultdisallow
     local urls
     ynh_handle_getopts_args "$@"
-    if [[ -n ${defaultdisallow:-} ]]; then
-        defaultdisallow=",default_allow=False"
-    fi
 
     if [[ -n ${urls:-} ]]; then
         urls=",urls=['${urls//';'/"','"}']"
     fi
-    yunohost tools shell -c "from yunohost.permission import permission_add; permission_add('$app', '$permission' ${defaultdisallow:-} ${urls:-}, sync_perm=False)"
+    yunohost tools shell -c "from yunohost.permission import permission_create; permission_create('$app.$permission' ${urls:-}, sync_perm=False)"
```

Comment:
Shouldn't we use "add/remove" everywhere instead? Like "user create" is the only place for that and should probably want to get ride of that (or on the contrary user "create/remove" for domains/other things?)

Reply:
Hmyeah we discussed this in length with ljf the other day ...

To me (and this is confirmed by [https://ux.stackexchange.com/questions/43174/update-vs-modify-vs-change-create-vs-add-delete-vs-remove]) : 
- creating/deleting is about creating something from scratch (or removing it from existence entirely)
- add/remove is about manipulating entities that already exist and adding/removing them to/from a system

So when talking about `user create` vs `user add`, it all depends wether you think of it as "creating an account" (the account doesn't exist yet, it is created) or "adding an (existing) human being to the system" (the human being already exist in real life). Personally I'm okay with the current `user create/remove` because I do think of it as creating an account. (The button on the webadmin says `+ New user` which is consistent with the idea of creating a new account as well.)  But I understand that other people might prefer thinking about it in term of "adding a human being" ... 

When talking about `domain create` vs `domain add`, it all depends wether you think of it as "creating a domain from scratch" (the domain doesn't exist yet anywhere on the internet) or "adding an (existing) domain name to the system" (the domain already exist on your registrar API). In the general case, I naively think more about "adding it" because I already created it on the registrar API ... but it's more ambiguous for the case of nohost.me / noho.st / ynh.fr which are created automatically when you add them ...

Similarly, you `create/delete` a backup archive (it didn't exist yet) and you `add/remove` a service (that you created in systemd first for example)

In the context of permission, to me the object really is `created/deleted` in that context (it doesn't exist yet), and then you `add/remove` groups / users to the permission and all the wording is consistent (and we really want to avoid `add` being ambiguously associated with the idea of "initializing" a new permission vs. allowing additional groups/members)

At least that's my thoughts on this ¯\\\_(ツ)\_/¯

Reply:
Thanks for the clarification. I'm too for the idea to be more precise about these both term. So yes, we should "create" a permission with the helper. And after we add a user/group in a specific permission.